.npmrc

```diff
@@ -0,0 +1,5 @@
+# .npmrc
+auto-install-peers=true
+shamefully-hoist=true
+strict-peer-dependencies=true
+shared-workspace-lockfile=false
```

Comment:
ensure these are correct and not copied from other repositories. over here we are not using workspaces.

Reply:
I removed `shared-workspace-lockfile=false `